src/Apps/Collect/CollectMediumMetadata.ts

```diff
@@ -0,0 +1,65 @@
+export function getMetadataForMedium(medium) {
+  let title = ""
+  let mediumDescription = ""
+
+  switch (medium) {
+    case "painting":
+      title = "Paintings"
+      mediumDescription = "250,000 paintings"
```

Comment:
Curious why we decided to go with hardcoded values for this, are these something we will need to update periodically?  Or are they not available/expensive to get counts for?

Reply:
Yeah, too expensive and computationally costly to compute solely for this use-case . We won't need to update them often.